.github/workflows/github-analytics-daily.yml

```diff
@@ -0,0 +1,64 @@
+name: GitHub Analytics Daily
+
+on:
+  schedule:
```

Comment:
```suggestion
  pull:
  	- .github/workflows/github-analytics-daily.yml
  schedule:
```

Reply:
No this will not be on every PR only when someone changes :
``.github/workflows/github-analytics-daily.yml``
also please add ``tools/analytics/github_analyze.py`` this way it will be triggered when either of these changes